### ISSUE TITLE
Add spar / alignment rods to aileron surfaces when printing split models.

### ISSUE THREAD

**Issue by mmosca** [StephenCarlson/MiniHawk-VTOL#49]
When printing the split model model for the wing you end up with the aileron surface split in half, with nothing but glue holding the pieces together.
For printers with not enough Z height it would be nice to have the ailerons print as one piece, or provide some way to keep the two pieces aligned either with a carbon rod or small spar.

This would ensure alignment and reinforce the joint.

**Comment by P-Nishanth** [comment]
I agree. If the wing is printed in parts, the aileron section will be very weak.
